mne/viz/tests/test_ica.py

```diff
@@ -72,24 +72,37 @@ def test_plot_ica_components():
         for components in [0, [0], [0, 1], [0, 1] * 2, None]:
             ica.plot_components(components, image_interp='bilinear',
                                 colorbar=True, **fast_test)
+        plt.close('all')
 
         # test interactive mode (passing 'inst' arg)
-        plt.close('all')
         ica.plot_components([0, 1], image_interp='bilinear', inst=raw, res=16)
-
         fig = plt.gcf()
-        ax = [a for a in fig.get_children() if isinstance(a, plt.Axes)]
-        lbl = ax[1].get_label()
-        _fake_click(fig, ax[1], (0., 0.), xform='data')
+
+        # test title click
+        # ----------------
+        lbl = fig.axes[1].get_label()
+        ica_idx = int(lbl[-3:])
+        titles = [ax.title for ax in fig.axes]
+        title_pos_midpoint = (titles[1].get_window_extent().extents
+                              .reshape((2, 2)).mean(axis=0))
+        # first click adds to exclude
+        _fake_click(fig, fig.axes[1], title_pos_midpoint, xform='pix')
+        assert_true(ica_idx in ica.exclude)
```

Comment:
For future reference with `pytest` use you can now just do:
```
assert ica_idx in ica.exclude
```
And get a nice error message if it fails, etc. No need to change it here, though.